perfkitbenchmarker/providers/gcp/flags.py

```diff
@@ -35,3 +35,10 @@
     'https://cloud.google.com/sdk/gcloud/reference/compute/disks/create')
 flags.DEFINE_string('gce_network_name', None, 'The name of an already created '
                     'network to use instead of creating a new one.')
+flags.DEFINE_list('vm_metadata_from_string', [],
```

Comment:
Was this meant to be vm_metadata_from_file? That's what I see in your changes to gce_virtual_machine.py and gce_virtual_machine_test.py.